server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java

```diff
@@ -256,11 +313,47 @@ void testEnvironmentVariablesNotLoadedWithInvalidYamlContent() {
         }
     }
 
+    @Test
+    void testEnvironmentVariablesNotLoadedWithInvalidJsonContent() {
```

Comment:
testEnvironmentVariablesNotLoadedWithInvalidJsonContent() and testEnvironmentVariablesNotLoadedWithInvalidYamlContent() could be converted into parameterized test

```
@ParameterizedTest(name = "testEnvironmentVariablesNotLoadedWithInvalidContent(): {index}")
@MethodSource("getInvalidContent")
void testEnvironmentVariablesNotLoadedWithInvalidContent(List<Resource> resources) {
...
}

private static Stream<List<Resource>> getInvalidContent() throws JsonProcessingException {
      return Stream.of(resourcesWithInvalidYamlContent(), resourcesWithInvalidJsonContent());
}

```

Reply:
Fixed by Irina